var/spack/repos/builtin/packages/scine-qcmaquis/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class ScineQcmaquis(CMakePackage):
+    """Scine QCMaquis DMRG Solver"""
+
+    homepage = "https://scine.ethz.ch/download/qcmaquis"
+    git = "https://github.com/qcscine/qcmaquis.git"
+
+    maintainers("adam-grofe")
+
+    version("master", branch="master")
+    version("3.1.3", branch="release-3.1.3")
+    version("3.1.2", branch="release-3.1.2", preferred=True)
+    variant(
+        "blas",
+        values=("openblas", "mkl"),
+        default="openblas",
+        description="Which blas library to use.",
+    )
+    variant(
+        "symmetries",
+        default="SU2U1PG,TwoU1PG",
+        description='Wave functions symmetries to compile (e.g. "SU2U1PG,TwoU1PG")',
+        values=("U1", "TwoU1", "TwoU1PG", "NU1", "Z2", "SU2U1", "SU2U1PG", "U1DG", "NONE"),
```

Comment:
Should the value(s) be validated?  For example, what if someone included `NONE`?

Reply:
According the QcMaquis CMakeLists.txt, "TwoU1PG and SU2U1PG are built in any case."